### DISCUSSION TITLE
Host to Linux

### DISCUSSION THREAD

**Discussion by GlobalFest** [dchapyshev/aspia#294] [Ideas]
Добрый день, подскажите а будет ли aspia host на линукс машинах? было бы очень удобно иметь такую возможность.

**Comment by pluralplay** [comment]
Дмитрий, давайте вы создадите донатный сбор, а мы вам задонатим на создание хоста на линукс? :)

**Comment by marcoafo** [comment]
(I don't speak Russian I used Google Translator for the questions above.)

I've already gave some work on that, for another project. It's a much more problematic issue. While it's possible to do in X11, in Wayland that's way more complicated. Perhaps one of the reasons Wayland didn't fully take over X11. Another issue is that it would give the screenshot only for the graphical interface, the console is another thing.

Another option is to use the DRM directly to copy the monitor buffer, but that's complicated and when we go to GPU world it's another discussion.

Considering to limit it to X11 might be an option to integrate, and I would love to help (as I'm stuck in the MacOS issues trying to build that). But since X11 is somehow considered deprecated, I'm still waiting for the Wayland API to come to stable for screenshot.
